src/utils/components/WindowsDrivers/WindowsDrivers.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { FC, useCallback, useState } from 'react';
+
+import { V1VirtualMachine } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import Loading from '@kubevirt-utils/components/Loading/Loading';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { mountWinDriversToVM } from '@kubevirt-utils/resources/vm/utils/disk/drivers';
+import { Alert, AlertVariant, Checkbox, Flex, FlexItem } from '@patternfly/react-core';
+
+import { removeWindowsDrivers, useDriversImage } from './utils';
+
+type WindowsDriversProps = {
+  vm: V1VirtualMachine;
+  updateVM: (vm: V1VirtualMachine) => void | Promise<void | V1VirtualMachine>;
+};
+
+const WindowsDrivers: FC<WindowsDriversProps> = ({ vm, updateVM }) => {
+  const { t } = useKubevirtTranslation();
+  const [error, setError] = useState<Error>(undefined);
+  const [loading, setLoading] = useState(false);
+
+  const [driversImage, driversImageLoading] = useDriversImage();
+
+  const onChange = useCallback(
+    async (checked: boolean) => {
+      setLoading(true);
+
+      try {
+        let newVM;
+        if (checked) {
+          newVM = await mountWinDriversToVM(vm);
+        } else {
+          newVM = removeWindowsDrivers(vm);
+        }
+
+        await updateVM(newVM);
```

Comment:
await updateVM(checked? await mountWinDriversToVM(vm) :removeWindowsDrivers(vm) );

Can see if this working properly?